addons/event/models/event_registration.py

```diff
@@ -282,6 +282,10 @@ def write(self, vals):
         if confirming:
             to_confirm._update_mail_schedulers()
 
+        if vals.get('state') == 'done':
+            message = _("Attended on %s", format_date(env=self.env, value=fields.Datetime.now(), date_format='dd/MM/Y'))
```

Comment:
Still the same issue with the format since we pass a hard-coded one. Instead use one of `“full”, “long”, “medium”, or “short”` (defined in babel). "short" should work here.

And this way if in the en_us it will be m/d/Y, en_uk dd/mm/yyyy, ...
maybe to check with PO whether we want to align with locale or hardcode some format. I guess it's not consistent throughout the codebase anyway
```suggestion
            message = _("Attended on %s", format_date(env=self.env, value=fields.Datetime.now(), date_format='short'))
```